blog/admin.py

```diff
@@ -0,0 +1,34 @@
+from django.contrib.auth.admin import UserAdmin
+from .models import User, Post, Commentary
+from django.contrib import admin
+from django.contrib.auth.models import Group
```

Comment:
Order and group your imports according to PEP-8. Add a blank line between different import groups. Use Absolute imports instead of relative ones